config/clients/python/template/exceptions.mustache

```diff
@@ -5,6 +5,17 @@
 import six
 
 
+# Specifc FGA header to be parsed
+X_RATELIMIT_LIMIT = 'x-ratelimit-limit'
+X_RATELIMIT_REMAINING = 'x_ratelimit_remaining'
+X_RATELIMIT_RESET = 'x_ratelimit_reset'
+FGA_REQUEST_ID = 'fga-request-id' 
+FGA_QUERY_DURATION_MS = 'fga-query-duration-ms'
+OPENFGA_AUTHORIZATION_MODEL_ID = 'openfga_authorization_model_id'
+INTERESTING_HEADERS = [X_RATELIMIT_LIMIT, X_RATELIMIT_REMAINING, X_RATELIMIT_RESET,
```

Comment:
😆 I like the term, but maybe we can use something more indicative? E.g. `RESPONSE_HEADERS_TO_KEEP`